src/components/reviewCard.jsx

```diff
@@ -1,19 +1,39 @@
 import { useParams } from "react-router-dom";
 import { useState, useEffect } from "react";
-import { fetchReviewById } from "../api";
+import { fetchReviewById, patchReviewVotes } from "../api";
 
 const ReviewCard = () => {
     const { review_id } = useParams();
     const [review, setReview] = useState({});
     const [reviewLoading, setreviewLoading] = useState(true);
+    const [reviewVotes, setReviewVotes] = useState(0);
+    const [incVotes, setIncVotes] = useState(0);
 
     useEffect(() => {
         fetchReviewById(review_id).then((data) => {
             setReview(data);
+            setReviewVotes(data.votes);
             setreviewLoading(false);
         });
     }, [review_id]);
 
+    useEffect(() => {
+        patchReviewVotes(review_id, incVotes)
+            .then((data) => {
+                setReviewVotes(data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    }, [review_id, incVotes]);
+
+    const upVoteHandler = (effect) => {
+        setIncVotes(1);
+    };
+    const downVoteHandler = (effect) => {
+        setIncVotes(-1);
+    };
```

Comment:
you should be optimistically rendering the votes so if the user has slow internet and upvotes it will still change their vote straight away but will catch an error if one occurs and then deal with the vote going back down